proteus/mprans/RANS3PF.h

```diff
@@ -1050,28 +1058,28 @@ namespace proteus
         duc_du = u/(uc+1.0e-12);
         duc_dv = v/(uc+1.0e-12);
         duc_dw = w/(uc+1.0e-12);
-        double fluid_velocity[3]={u,v,w}, solid_velocity[3]={u_s,v_s,w_s};
+        double fluid_velocity[3]={uStar,vStar,wStar}, solid_velocity[3]={uStar_s,vStar_s,wStar_s};
         double new_beta =   closure.betaCoeff(1.0-phi_s,
                                           rho,
                                          fluid_velocity,
                                            solid_velocity,
                                             viscosity);
         //new_beta/=rho;
-        mom_u_source +=  (1.0 - phi_s)*new_beta*( (u - u_s) - nu_t*gradC_x/closure.sigmaC_ );
-        mom_v_source +=  (1.0 - phi_s)*new_beta*( (v - v_s)- nu_t*gradC_y/closure.sigmaC_ );
-        mom_w_source +=  (1.0 - phi_s)*new_beta*( (w - w_s)- nu_t*gradC_z/closure.sigmaC_ );;
+        mom_u_source +=  (1.0 - phi_s)*new_beta*( (u - uStar_s) - nu_t*gradC_x/closure.sigmaC_ );
```

Comment:
Does this mean that the calculation is updated? Or just the naming

Reply:
It's changed slightly to use the extrapolated/lagged u_s.  After looking at the way we were handling the momentum exchange, I changed a few things to make each sub-step linear and ensure conservation of momentum in the exchange. There are several ways to keep it conservative.  We may need to  return to this issue.